UPort/JWT/JWTTools.swift

```diff
@@ -76,6 +76,60 @@ public struct JWTTools
         }
     }
 
+    public static func create(payload: [String: Any],
+                              issuerDID: String,
+                              signer: Signer,
+                              expiresIn: Int64,
+                              completionHandler: @escaping (_ fullJWT: String?, _ error: Error?) -> Void)
+    {
+        // Construct header and convert to  base64
+        let headerArgs: [String: Any] = ["typ": "JWT", "alg": "ES256K-R"]
+        let headerString = JWTTools.dictionaryToBase64(dict: headerArgs)
+        let headerBase64Url = JWTTools.base64ToBase64Url(base64String: headerString)
+
+        // Fill out payload with iss, iat, and exp
+        var filledOutPayload = payload
+        filledOutPayload["iat"] = Int64(now().timeIntervalSince1970)
+        if filledOutPayload["exp"] == nil { filledOutPayload["exp"] = Int64(now().timeIntervalSince1970) + expiresIn }
+        filledOutPayload["iss"] = issuerDID
+        // Convert filled out payload to base64
+        let payloadBase64 = JWTTools.dictionaryToBase64(dict: filledOutPayload)
+        let payloadBase64Url = JWTTools.base64ToBase64Url(base64String: payloadBase64)
+
+        // Join the header/payload and base64 encode for signing
+        let signingInput = [headerBase64Url, payloadBase64Url].joined(separator: ".")
+        let signingData = signingInput.data(using: .utf8)
+        let signingInputBase64 = signingData?.base64EncodedString()//.replacingOccurrences(of: "=", with: "")
+        let signingInputBase64Url = JWTTools.base64ToBase64Url(base64String: signingInputBase64!)
+
+        // Sign jwt
+        signer.signJWT(rawPayload: signingInputBase64Url) { (sig, error) in
+            guard error == nil else
+            {
+                completionHandler(nil, error)
+                return
+            }
+            do
+            {
+                if sig != nil
+                {
+                    let rData = try? (sig!["r"] as? String)?.decodeBase64()
+                    let sData = try? (sig!["s"] as? String)?.decodeBase64()
+                    var vNum = sig!["v"] as? Int
+                    let vData = UnsafeBufferPointer(start: &vNum, count: 1)
+                    var rsv = Data()
+                    rsv.append(rData!!)
+                    rsv.append(sData!!)
+                    rsv.append(vData)
+                    let sigBase64 = rsv.base64EncodedString().replacingOccurrences(of: "=", with: "")
```

Comment:
How about moving `.replacingOccurrences(of: "=", with: "")` to `JWTTools.base64ToBase64Url()`

Reply:
sounds like a good idea